app/views/support/providers/locations/check_multiple/show.html.erb

```diff
@@ -0,0 +1,24 @@
+<% page_title = t("support.providers.multiple_locations.check") %>
+<%= content_for :page_title, page_title %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+    <%= form_with(model: @parsed_csv_school_form, url: support_recruitment_cycle_provider_locations_multiple_check_path(provider_code: @provider.provider_code, recruitment_cycle_year: @provider.recruitment_cycle_year), method: :put, local: true) do |f| %>
+      <legend class="govuk-fieldset__legend govuk-fieldset__legend--l"> <span class="govuk-caption-l"><%= "Add locations - #{@provider.decorate.name_and_code}" %></span>
+        <h1 class="govuk-fieldset__heading govuk-!-margin-bottom-3"><%= t("support.providers.multiple_locations.check") %> </h1> </legend>
+        <% @school_details.each_with_index do |school_detail, index| %>
+          <%= govuk_summary_card(title: "Location #{index + 1}") do |card|
+                card.with_action { govuk_link_to("Change", "https://trello.com/c/BiPYisS2/1109-bulk-uploads-changing-locations-on-csv-confirm-page") }
+                card.with_summary_list(rows: [{ key: { text: "Name" }, value: { text: school_detail.location_name } },
+                                              { key: { text: "Unique reference number (URN)" }, value: { classes: ("govuk-hint" if school_detail.urn.blank?), text: (school_detail.urn.blank?) ? "Not entered" : school_detail.urn } },
+                                              { key: { text: "Address" }, value: { text: school_detail.decorate.full_address_on_seperate_lines } }])
+              end %>
+      <% end %>
+      <%= f.govuk_submit("Add locations") %>
+    <% end %>
+
+    <p class="govuk-body">
+      <%= govuk_link_to(t("cancel"), support_recruitment_cycle_provider_locations_path) %>
```

Comment:
@simonwhatley should this cancel button (from the confirm page) take you to the page where the `Add multiple locations` button is? I felt like it should have taken me to the `/new` page.

Reply:
<img width="568" alt="image" src="https://user-images.githubusercontent.com/50492247/225724302-4098489d-d682-476a-a22c-e22a111e77d7.png">
This is the page it takes you to currently.